index.test.ts

```diff
@@ -85,19 +85,25 @@ test("handles events", async () => {
             now: "2020-01-01T01:01:01Z"
         }
     ]
-
     const db = createSnowflakeMock(snowflakeAccount)
 
     await snowflakePlugin.setupPlugin?.(meta)
-    await snowflakePlugin.exportEvents?.(events, meta)
+    for (let i = 0; i < 3; i++) { // to have >1 files to copy over
+        await cache.expire('lastRun', 0)
```

Comment:
Non-blocking

Rather than using the cache here, we can set the time to a later date and run again. fine with this though